hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -1004,7 +1004,7 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status,
         keyInfo.getModificationTime(),
         keyInfo.getModificationTime(),
         status.isDirectory() ? (short) 00777 : (short) 00666,
-        owner,
+        StringUtils.defaultIfEmpty(keyInfo.getOwnerName(), owner),
         owner,
```

Comment:
Same. `group` should also have the same treatment?

```suggestion
        StringUtils.defaultIfEmpty(keyInfo.getOwnerName(), owner),
        StringUtils.defaultIfEmpty(keyInfo.getOwnerName(), owner),
```